test/lib/cql_assertions.cc

```diff
@@ -38,7 +38,7 @@ rows_assertions::is_empty() {
     auto row_count = rs.size();
     if (row_count != 0) {
         auto&& first_row = *rs.rows().begin();
-        fail(format("Expected no rows, but got {:d}. First row: {}", row_count, to_string(first_row)));
+        fail(format("Expected no rows, but got {:d}. First row: {}", row_count, fmt::to_string(first_row)));
```

Comment:
I don't understand this patch. After you supposedly fixed the to_string() function for ranges in the previous patch, now you don't use it and use fmt::to_string() instead. Why?

Reply:
Most importantly, what happens if you DONT do this patch? Does to_string(first_row) do something wrong and fmt::to_string(first_row) fix it? If the answer is yes, we have a big problem because developers are very likely to type to_string() instead of fmt::to_string(), and if it compiles - they won't realize they made a misttake.
If the answer is no, then why is this patch needed?

Reply:
> I don't understand this patch. After you supposedly fixed the to_string() function for ranges in the previous patch, now you don't use it and use fmt::to_string() instead. Why?

This patch removes the to_string definition from utils/to_string.hh (which makes its name archaic to say the least :)).
I didn't want to do this in a single patch, therefore the separation to patches.
I'll add a FIXME comment in the first patch about this to make the intention clearer.

> Most importantly, what happens if you DONT do this patch? Does to_string(first_row) do something wrong and fmt::to_string(first_row) fix it? If the answer is yes, we have a big problem because developers are very likely to type to_string() instead of fmt::to_string(), and if it compiles - they won't realize they made a misttake. If the answer is no, then why is this patch needed?

We want to implement as little as possible and have a concise implementation.
I'd like the fmt library to do the heavy lifting in this case and use its magic to get to the right formatter.